### ISSUE TITLE
Can't install due to "Failed at the extract-file-icon@0.3.1 rebuild script." error.

### ISSUE THREAD

**Issue by payitforwardnow** [wexond/browser-base#343]
**Bug description**
Can't install using "yarn"

**To Reproduce**
All I did was load the project into Visual Studio Code and type "yarn" in the terminal window. 

**Expected behavior**
I expect the package to be installed

**Screenshots**
No need.

**Details:**
 - Operating System: MacOS High Sierra 10.13.6
 - Wexond version: 3.1.0
 - Last known working Wexond version: unknown

**Additional context**
Terminal log files below. Sorry I put the entire log but honestly I don't know what is important and what is not. Thank you in advance for your time and consideration. FYI - Was able to compile the official release version 3.1.0 in both win and mac; however, I was not able to run it in developer mode ("yarn dev"). Not sure it that's related. Thought I'd mention it. 

Mickey-2:desktop-master 2 popeye$ yarn
yarn install v1.17.0
info No lockfile found.
warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
[1/4] 🔍  Resolving packages...
warning electron > electron-download > nugget > progress-stream > through2 > xtend > object-keys@0.4.0: 
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > styled-components@5.0.0-beta.8-groupsizefix" has unmet peer dependency "react-is@>= 16.8.0".
warning "styled-components > stylis-rule-sheet@0.0.10" has unmet peer dependency "stylis@^3.5.0".
[4/4] 🔨  Building fresh packages...
[1/8] ⠁ electron
[-/8] ⠁ waiting...
[3/8] ⠁ extract-file-icon
[7/8] ⠁ node-window-manager
error /Volumes/Projects/Electron/Wexond/desktop-master 2/node_modules/extract-file-icon: Command failed.
Exit code: 1
Command: npm run rebuild
Arguments: 
Directory: /Volumes/Projects/Electron/Wexond/desktop-master 2/node_modules/extract-file-icon
Output:
> extract-file-icon@0.3.1 rebuild /Volumes/Projects/Electron/Wexond/desktop-master 2/node_modules/extract-file-icon
> node-gyp rebuild

  CXX(target) Release/obj.target/addon/lib/addon.o
clang: error: no such file or directory: '2/node_modules/node-addon-api'
make: *** [Release/obj.target/addon/lib/addon.o] Error 1
gyp ERR! build error 
gyp ERR! stack Error: `make` failed with exit code: 2
gyp ERR! stack     at ChildProcess.onExit (/usr/local/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:194:23)
gyp ERR! stack     at ChildProcess.emit (events.js:210:5)
gyp ERR! stack     at Process.ChildProcess._handle.onexit (internal/child_process.js:272:12)
gyp ERR! System Darwin 17.7.0
gyp ERR! command "/usr/local/bin/node" "/usr/local/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
gyp ERR! cwd /Volumes/Projects/Electron/Wexond/desktop-master 2/node_modules/extract-file-icon
gyp ERR! node -v v12.13.0
gyp ERR! node-gyp -v v5.0.5
gyp ERR! not ok 
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! extract-file-icon@0.3.1 rebuild: `node-gyp rebuild`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the extract-file-icon@0.3.1 rebuild script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm WARN Local package.json exists, but node_modules missing, did you mean to install?

**Comment by payitforwardnow** [comment]
BTW - I don't know what that last warning was about. The node_modules folder exists. May not be an issue.

**Comment by payitforwardnow** [comment]
I tried removing the node_modules folder.  Same error.

**Comment by payitforwardnow** [comment]
Also tried deleting the node_modules folder and the "npm install" just in case yarn was having a problem. Same error.

**Comment by payitforwardnow** [comment]
Additional info:
node-gyp    v6.0.0
node            v12.13.0 
npm             6.12.0 
yarn             1.17.0

Also saw an error: no such file or directory: '2/node_modules/node-addon-api' could this be related to a space in the folder path???? That would be strange.

**Comment by payitforwardnow** [comment]
Resolved: Change all spaces in project path to '-' (or something else or simply delete them). Not sure why the scripts don't like spaces but it appears spaces are a choking hazard. 

What a PITA!!! After all these years I can't believe this is the first time a space in the project path has bitten me.  Apparently I unzipped the desktop-master zip twice and it added " 2" on the end of the folder. 

I hope all the time I wasted, up until I added to this post, will help someone else from making the same mistake.
